server/api/checkout.js

```diff
@@ -0,0 +1,34 @@
+const stripe = require('stripe')('sk_test_4eC39HqLyjWDarjtT1zdp7dc')
+const express = require('express')
+// const router = require('express').Router()
+const app = express()
+app.use(express.static('public'))
+
+const YOUR_DOMAIN = 'http://localhost:8080'
+
+app.post('/create-checkout-session', async (req, res) => {
```

Comment:
This route should have token checking for the order's userId and the token's user ID. Also, you _may_ have to store some kind of data from the stripe session in your database (not positive on this), and you definitely will need to get YOUR_DOMAIN in some other way because this won't work on your deployed site as it is